backend/auth/graphql/resolver.go

```diff
@@ -0,0 +1,78 @@
+package graphql
+
+import (
+	"context"
+
+	"github.com/jinzhu/gorm"
+	"github.com/stamhoofd/stamhoofd/backend/auth/models"
+	"github.com/stamhoofd/stamhoofd/backend/auth/service"
+)
+
+type Resolver struct {
+	DB *gorm.DB
+}
+
+func (r *Resolver) Mutation() MutationResolver {
+	return &mutationResolver{r}
+}
+func (r *Resolver) Query() QueryResolver {
+	return &queryResolver{r}
+}
+
+type mutationResolver struct{ *Resolver }
+
+var user = &User{
+	ID:    "1",
+	Email: "name@example.com",
+}
+var loggedInResponse = &LoggedInResponse{
+	User:  user,
+	Token: "token",
+}
+var resetPasswordResponse = &ResetPasswordResponse{
+	Email: user.Email,
+}
+
+func NewUser(user *models.User) *User {
+	return &User{
+		ID:    user.ID.String(),
+		Email: user.Email,
+	}
+}
+
+func (r *mutationResolver) Register(ctx context.Context, email, password string) (*User, error) {
+	user, err := service.Register(r.DB, email, password)
+	if err != nil {
+		return nil, err
+	}
+	return NewUser(user), nil
+}
+
+func (r *mutationResolver) RegisterConfirm(ctx context.Context, email, token string) (*LoggedInResponse, error) {
+	user, err := service.RegisterConfirm(r.DB, email, token)
+	if err != nil {
+		return nil, err
+	}
+	return &LoggedInResponse{
+		User:  NewUser(user),
+		Token: user.AuthenticationToken,
```

Comment:
Token should contain `RefreshToken`, `AccessToken` and some expiry date (+ see other comments about storing it in a separate model).

Reply:
Re: others. This is just temporary. We have to think about how we will implement tokens. This is just to get going. It's very rough around the edges, but this database "design" is not **at all** how it will be, that's also why the database is SQLite in memory right now and not PostgreSQL/MySQL.